src/backend/cgelem.c

```diff
@@ -5443,4 +5448,89 @@ void postoptelem(elem *e)
     }
 }
 
+/***********************************
+ * Rewrite rvalues of complex numbers to pairs of floating point numbers.
+ */
+static elem *elToPair(elem *e)
+{
+    switch (e->Eoper)
+    {
+        case OPvar:
+        {
+#if 0
```

Comment:
I intend to turn it on in the next PR. Otherwise too many changes at once, making regressions harder to find.